src/syscheckd/db/include/fimDB.hpp

```diff
@@ -222,7 +215,6 @@ constexpr auto FIM_VALUE_START_CONFIG_STATEMENT
     //TO DO
 };
 
-
 class FIMDB
```

Comment:
this can be final to avoid inheritance?

Reply:
Right now we mocked FIMDB by inheriting from it in our mock class. Maybe it's not necessary to do that. This issue will be addressed in the following [issue](https://github.com/wazuh/wazuh/issues/11332)